connect/src/com/telenor/connect/id/Claims.java

```diff
@@ -8,6 +8,7 @@
 
     public static final String OPENID = "openid";
     public static final String PROFILE = "profile";
```

Comment:
I might have misinterpreted the docs?:

> The available claims to request have the same values as the available scope values

http://docs.telenordigital.com/apis/connect/id/authentication.html